driver/bpf/filler_helpers.h

```diff
@@ -859,17 +893,15 @@ static __always_inline int __bpf_val_to_ring(struct filler_data *data,
 
 				if (!data->curarg_already_on_frame) {
 					volatile u16 read_size = dpi_lookahead_size;
+					int rc;
```

Comment:
I just realized that when `BPF_FORBIDS_ZERO_ACCESS` is defined and `read_size` is 0, `rc` is not set, should we initialize it?
```suggestion
					int rc = 0;
```